### PR TITLE
Documentation slackbot via `langchain`, `llama_index` and `OpenAI` integrations

### PR DESCRIPTION
**Pull request by htahir1**

### CODE REVIEW

File: langchain-llamaindex-slackbot/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Playground for docs-chat
+
+## Installation
```
Comment (htahir1) [inline langchain-llamaindex-slackbot/README.md:3]: This needs some love and id add a requirements.txt to make things easier

File: langchain-llamaindex-slackbot/src/.github/workflows/fly.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: Fly Deploy
```
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/.github/workflows/fly.yml:1]: delete , this isnt used anymore

File: langchain-llamaindex-slackbot/src/Procfile [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Modify this Procfile to fit your needs
+bot: zenml connect --url $ZENML_SERVER_URL --username $ZENML_USERNAME --password $ZENML_PASSWORD \
```
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/Procfile:2]: delete not used anymore

File: langchain-llamaindex-slackbot/src/__init__.py
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/__init__.py:1]: add licence

File: langchain-llamaindex-slackbot/src/fly.toml
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/fly.toml:1]: delete not used anywhere

File: langchain-llamaindex-slackbot/src/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+# Define environment variable
+ENV SLACK_APP_TOKEN $SLACK_APP_TOKEN
+ENV SLACK_BOT_TOKEN $SLACK_BOT_TOKEN
+ENV OPENAI_API_KEY $OPENAI_API_KEY
+ENV PIPELINE_NAME $PIPELINE_NAME
+ENV ZENML_SERVER_URL $ZENML_SERVER_URL
+ENV ZENML_USERNAME $ZENML_USERNAME
+ENV ZENML_PASSWORD $ZENML_PASSWORD
+
```
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/Dockerfile:26]: maybe add ZENML_ANALYTICS_OPT_IN as false?

File: langchain-llamaindex-slackbot/src/query_data.py
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/query_data.py:1]: Id delete this as its not used?

File: langchain-llamaindex-slackbot/src/schemas.py
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/schemas.py:1]: Id delete this as its not used?

File: langchain-llamaindex-slackbot/src/callback.py
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/callback.py:1]: Id delete this as its not used?

File: langchain-llamaindex-slackbot/src/templates/index.html
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/templates/index.html:1]: Id delete this as its not used?

File: langchain-llamaindex-slackbot/src/web_app.py
Comment (htahir1) [inline langchain-llamaindex-slackbot/src/web_app.py:1]: Id delete this as its not used?

**Review by htahir1 — COMMENTED** [review]
Initial review

File: langchain-llamaindex-slackbot/README.md (hunk 1 above)
Comment (strickvl) [reply langchain-llamaindex-slackbot/README.md:65]: Added a bunch of things in there now.

**Pull request merged**